Yoast/wordpress-seo
# Update the styling of the toggles

This PR replaces https://github.com/Yoast/wordpress-seo/pull/14609

## Context


* Implement the new toggles everywhere.

## Summary


This PR can be summarized in the following changelog entry:

* Implement a new toggle design based on checkboxes instead of radio buttons.

## Relevant technical choices:

* Decided to make an `inverse` parameter for the `light_switch` function in `Yoast_Form`. You should set this to `true` when we store a negative value, for instance on `noindex-` and `disable-` type options.

## Test instructions

This PR can be tested by following these steps:

* Checkout this branch with the latest `develop` from the monorepo
* All options that have toggles should save, work and behave as expected.
* This includes: General - Features, Search Appearance - Content types
* Make sure the invert toggles work and also the toggles with longer labels

For CR:
* Make sure to also CR the changes on this (merged) PR https://github.com/Yoast/javascript/pull/649
* A CR of (part of) this code was already done in: https://github.com/Yoast/wordpress-seo/pull/14609, please check this too

## UI changes

## Documentation

## Quality assurance


Fixes #14290
